ecospheres_universe/feed_universe.py

```diff
@@ -34,7 +34,7 @@ class Organization(NamedTuple):
     id: str
     name: str
     slug: str
-    type: str
+    type: Optional[str]
```

Comment:
```suggestion
    type: str | None
```
I think it's the new Pythonic Way ™️ ?